rebar.config

```diff
@@ -13,13 +13,10 @@
     {gproc, "0.9.0"},
     {backoff, "1.1.6"},
     {clique, ".*", {git, "https://github.com/helium/clique.git", {branch, "develop"}}},
-    {grpcbox,
-        {git, "https://github.com/novalabsxyz/grpcbox.git",
-            {ref, "6180be0f4f4f199f824e60ae5c64ddd2daa5acff"}}},
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git", {branch, "mj/full-update"}}},
```

Comment:
Can we get this PR merged?

Reply:
working on it. https://github.com/novalabsxyz/grpcbox/pull/16